src/pages/Upload/UploadPage.jsx

```diff
@@ -1,79 +1,74 @@
 import React, { useState } from 'react';
-import {
-  Button,
-  Stack,
-  Typography,
-  Link,
-  InputAdornment,
-  FormControl,
-  InputLabel,
-  OutlinedInput,
-} from '@mui/material';
+import { Button, Dialog } from '@mui/material';
+import UploadIcon from '@mui/icons-material/Upload';
+import styles from './UploadPopUp.module.css';
```

Comment:
```suggestion
import styles from './UploadPage.module.css';
```
![image](https://user-images.githubusercontent.com/58793979/160736726-7240202f-7208-4722-bb19-6ad00532dd90.png)
I seem to be getting this error in the local environment, maybe try out this suggestion please.